airbyte-ci/connectors/connectors_qa/tests/integration_tests/test_documentation.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+
+from pathlib import Path
+
+import git
+import pytest
+from asyncclick.testing import CliRunner
+from connectors_qa.cli import generate_documentation
+
+DOCUMENTATION_FILE_PATH_IN_AIRBYTE_REPO = Path("docs/contributing-to-airbyte/resources/qa-checks.md")
+
+@pytest.fixture
+def airbyte_repo():
+    return git.Repo(search_parent_directories=True)
+
+@pytest.mark.asyncio
+async def test_generated_qa_checks_documentation_is_up_to_date(airbyte_repo, tmp_path):
+    # Arrange
```

Comment:
I really like this organization of test. I would love to see this reflected in the test naming too. I usually use "given_x_when_y_then_z". I hate Python snake case because it feels like it would be easier to read having something like `givenX_whenY_thenZ` but eh!